estate/models/estate_property.py

```diff
@@ -0,0 +1,42 @@
+from dateutil.relativedelta import relativedelta
+from odoo import models , fields
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "This is property tabel."
```

Comment:
```suggestion
    _description = "This is property tabel."
```
It should be more model specific, check codebase once